mappings/net/minecraft/world/biome/source/DefaultBiomeAccessStrategy.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_4546 net/minecraft/world/biome/source/DefaultBiomeAccessStrategy
```

Comment:
This strategy seems to use the same scale logic as the former voronoi scale biome layer -- maybe it should be named more specifically around that?

Reply:
`VoronoiBiomeScaleLogic` or what? I need some clue

Reply:
didn't mention because not sure of a good name. maybe just like `VoronoiBiomeAccess`?

Reply:
Hmm, I think Voroni algorithm is still in `ScaleLayer` class. This is only applying random fuzz and isn't exclusive to world gen (when entities/game logic gets biome, the call goes through this pipeline as well)

Reply:
1. The normal ScaleLayers don’t use voronoi tesselation.
2. Why did you spell “voronoi” as “voroni” lol

Reply:
And yeah voronoi tesselation scaling was applied for game logic getting biomes in 1.14

Reply:
done.